File: src/main/java/ru/bebriki/bebriki/models/Achievement.java
package ru.bebriki.bebriki.models;

import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDateTime;

@Builder
@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
@Entity
@Table(name = "achievements")
public class Achievement {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    @Column(name = "name")
    private String name;
    @Column(name = "description")
    private String description;
    @Column(name = "imageURL")
    private String imageURL;
    @Column(name = "date")
    private LocalDateTime date;
    @Column(name = "isDone")
    private Boolean isDone;
}

File: Front/neo-store-app/src/assets/scripts/models/achievement.ts
export default class Achievement {
    id: number;
    name: string;
    description: string;
    imageURL: string;
    date: string;
  
    constructor(
      id: number,
      name: string,
      description: string,
      imageURL: string,
      date: string
    ) {
      this.id = id;
      this.name = name;
      this.description = description;
      this.imageURL = imageURL;
      this.date = date;
    }
  }